plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/cwc/inline/InlineChatEditorHint.kt

```diff
@@ -91,7 +94,7 @@ class InlineChatEditorHint {
             location,
             HintManager.HIDE_BY_TEXT_CHANGE or HintManager.HIDE_BY_SCROLLING,
             0,
-            false,
+            true,
```

Comment:
what will this config `reviveOnEditorChange` impact

Reply:
reverted that, thanks for catching